solang-server/Cargo.toml

```diff
@@ -12,6 +12,7 @@ bytes = "0.5"
 solang = { git= "https://github.com/hyperledger-labs/solang" }
 dashmap = "3.5.1"
 futures = { version = "0.3", features = ["compat"] }
+inkwell = { version = "0.1.0-llvm10sample", features = ["target-webassembly", "llvm10-0"] }
```

Comment:
inkwell is not needed; solang depends on inkwell, but solang dependencies will pull that in. solang-server does not depend on inkwell directly.

Reply:
done